db/schema.rb

```diff
@@ -423,6 +423,7 @@
     t.boolean "allow_prompt_login", default: false
     t.boolean "signed_response_message_requested", default: false
     t.integer "ial2_quota"
+    t.boolean "liveness_checking_enabled"
```

Comment:
One thought, just an idea, not a requirement, what if we called this `liveness_checking_required` ? or `requires_liveness_checking` so it's slightly different than the global feature flag we have? I feel like having separate names would make it easier to debug

Reply:
yes i like!